src/NServiceBus.Core/NServiceBus.Core.csproj

```diff
@@ -28,6 +28,11 @@
     <PackageReference Include="Obsolete.Fody" Version="4.2.2" PrivateAssets="All" />
     <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="All" />
     <PackageReference Include="Particular.Licensing.Sources" Version="1.0.0-alpha0001" PrivateAssets="All" />
+    <PackageReference Include="System.Reflection.Metadata" Version="1.5.0-preview2-25405-01" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
+    <PackageReference Include="System.ValueTuple" Version="4.4.0-preview2-25405-01" />
```

Comment:
As mentioned during the sync meeting, referencing the preview version means we're using the same version of the assembly for both targets. Once the final bits are out, then we'll bump everything up.